CMPUT301F24bubble/bubbletracks
# Organizer custom notification message

Closes #32, #33, #34, and #25.

To test,:

- You need at least 2 devices. One for organizer and one for entrant.
-You can either follow the steps for the entrant to join the organizers event, or directly add your entrant device id to the lists (waitlist, enrolled, cancelled, invited) in your Firestore events document. 

- Go to the notifications activity the hosted events screen, and input your custom notification. Make sure to check off the type of entrant you want to send the notification to.
- Your entrant device should see the custom notification, and the notification document should be created.
